arch/cortex-m/src/syscall.rs

```diff
@@ -35,7 +35,7 @@ extern "C" {
 }
 
 // Space for 8 u32s: r0-r3, r12, lr, pc, and xPSR
```

Comment:
Why add 8 bytes? Shouldn't this just be one more `u32`, so 4 bytes? It would be good to also edit the comment above this

Reply:
Oops, apparently I can't do math.